src/PKSim.BatchTool/Presenters/BatchMainPresenter.cs

```diff
@@ -44,16 +53,18 @@ public void GenerateTrainingMaterial()
          start<IGenerateTrainingMaterialPresenter>();
       }
 
-      private void start<T>() where T : IInitializablePresenter<BatchStartOptions>
+      private void start<T>() where T : IBatchPresenter
       {
          var presenter = _applicationController.Start<T>();
-         _view.Hide();
+         View.Hide();
          presenter.InitializeWith(_startOptions);
       }
 
       public void GenerateProjectOverview()
       {
          start<IGenerateProjectOverviewPresenter>();
       }
+
+      public IView ActiveView => _startOptions.IsValid() ? _batchSilentView : BaseView;
```

Comment:
If you need to start a message loop with Application.Run then I don't have an alternative. 

I didn't investigate how this all works under the hood but inside main() if you don't call Application.Run then you don't need this view.